Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/FarePolicy.hs

```diff
@@ -223,6 +223,8 @@ fromTTypeFarePolicy handler BeamFP.FarePolicyT {..} = do
               farePolicyDetails,
               perMinuteRideExtraTimeCharge = perMinuteRideExtraTimeCharge,
               congestionChargeMultiplier = congestionCharge,
+              congestionChargePerMin = Nothing,
```

Comment:
Why Nothing here?

Reply:
because it's calculated dynamically, it won't be present in farePolicy table.

Reply:
Okay.